packages/amplication-data-service-generator/src/server/prisma/create-prisma-schema-fields.ts

```diff
@@ -0,0 +1,482 @@
+import {
+  Entity,
+  EntityField,
+  EnumDataType,
+  LookupResolvedProperties,
+  types,
+} from "@amplication/code-gen-types";
+import { camelCase } from "camel-case";
+import { pascalCase } from "pascal-case";
+import * as PrismaSchemaDSL from "prisma-schema-dsl";
+import * as PrismaSchemaDSLTypes from "prisma-schema-dsl-types";
+
+export const CUID_CALL_EXPRESSION = new PrismaSchemaDSLTypes.CallExpression(
+  PrismaSchemaDSLTypes.CUID
+);
+
+export const UUID_CALL_EXPRESSION = new PrismaSchemaDSLTypes.CallExpression(
+  PrismaSchemaDSLTypes.UUID
+);
+
+export const INCREMENTAL_CALL_EXPRESSION =
+  new PrismaSchemaDSLTypes.CallExpression(PrismaSchemaDSLTypes.AUTO_INCREMENT);
+
+export const NOW_CALL_EXPRESSION = new PrismaSchemaDSLTypes.CallExpression(
+  PrismaSchemaDSLTypes.NOW
+);
+
+export function createPrismaFields(
+  field: EntityField,
+  entity: Entity,
+  fieldNamesCount?: Record<string, number>
+):
+  | [PrismaSchemaDSLTypes.ScalarField]
+  | [PrismaSchemaDSLTypes.ObjectField]
+  | [PrismaSchemaDSLTypes.ObjectField, PrismaSchemaDSLTypes.ScalarField] {
+  return createPrismaSchemaFieldsHandlers[field.dataType](
+    field,
+    entity,
+    fieldNamesCount
+  );
+}
+
+export const createPrismaSchemaFieldsHandlers: {
+  [key in EnumDataType]: (
+    field: EntityField,
+    entity: Entity,
+    fieldNamesCount?: Record<string, number>
+  ) =>
+    | [PrismaSchemaDSLTypes.ScalarField]
+    | [PrismaSchemaDSLTypes.ObjectField]
+    | [PrismaSchemaDSLTypes.ObjectField, PrismaSchemaDSLTypes.ScalarField];
+} = {
+  [EnumDataType.SingleLineText]: (
+    field: EntityField,
+    entity: Entity,
+    fieldNamesCount: Record<string, number> = {}
+  ) => {
+    const { dataType, name, properties } = field;
```

Comment:
Please clean up the unused properties from every function. The lint is also yelling at you because of the unused variables

Reply:
and also, choose whether to destructure all the properties you need,
or do not destructre at all and use `field.{property}`, then return the value without the `return` keyword